From b0221a692c6564aecad20b969e08d0e59f59a45b Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 27 Oct 2025 17:22:20 +0100
Subject: [PATCH] dt-bindings: fix warnings on access-controllers property for
 vivante GPU

In the stm32mpxxx device tree files, the access-controllers property has
been added to the Vivante GPU peripheral nodes so that the firewall
configuration is checked before using it. Document it.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I39643cdfed2191177f73211161a66715100bf6fc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/503461
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 Documentation/devicetree/bindings/gpu/vivante,gc.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/gpu/vivante,gc.yaml b/Documentation/devicetree/bindings/gpu/vivante,gc.yaml
index 589cf4c31de6..42f86e0a1936 100644
--- a/Documentation/devicetree/bindings/gpu/vivante,gc.yaml
+++ b/Documentation/devicetree/bindings/gpu/vivante,gc.yaml
@@ -52,6 +52,9 @@ properties:
   contiguous-area:
     maxItems: 1
 
+  access-controllers:
+    maxItems: 1
+
 required:
   - compatible
   - reg
-- 
2.53.0

